src/Components/Button/Button.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 
+import styled from 'styled-components';
+
 export default function Button() {
   return (
-    <div>
-      <div>버튼!</div>
-    </div>
+    <ButtonWrapper backgroundColor color border>
+      <i /> 버튼
+    </ButtonWrapper>
   );
 }
+
+const ButtonWrapper = styled.button`
+  margin-top: 10px;
+  padding: 12px 0;
+  font-size: 14px;
+  background-color: ${props => props.backgroundColor && '#FFFFFF'};
+  color: ${props => props.color && '#222222'};
+  border: ${props => props.border && '1px solid #222222'};
+
```

Comment:
스타일상에는 조건부로 스타일링을 하고 있는데 ButtonWrapper를 보니까 무조건 true로 값을 전달해주고 있는 것 같습니다!
조건부스타일링을 할 필요가 없어보이네요 지금 로직상으로는!
혹시 나중에  Button 컴포넌트에서 props를 받으실 예정이신가요?
그리고  css의 value값안에서 조건을 넣고 계신데 이러면 만약 props값이 false라면 `background-color: undefined;` 라는 결과가 나올수도 있어보이네요!

```suggestion
  ${props => props.backgroundColor && 'background-color: #FFFFFF'};
```
이런식으로 하는게 좀 더 나아보입니다!

Reply:
나머지부분도 마찬가지입니다!

Reply:
그리고 모두 props받아와서쓰는거니까 한꺼번에 props받아와서 쓰는게 간편해보이면  
```suggestion
${({backgroundColor, color, border}) => css`
  ${backgroundColor && "background-color:#222222"};
  ${color && "color:#222222"};
  ${border && "border:#222222"};
`}
```
요건 어느부분이 더 좋은지는 취향차이입니다!

Reply:
리뷰 감사합니다. 현재는 중복되서 사용되지 않아 기본 값으로 세팅해두었습니다. 추후 조건부로 스타일링 혹은 props를 사용하게 되면 이 부분 유의하여 코드 작성해보도록하겠습니다.